docs/en/ingest-management/tab-widgets/install-layout.asciidoc

```diff
@@ -15,7 +15,7 @@ Log files for {beats} shippersfootnote:lognumbering[]
 Shell wrapper installed into PATH
 
 You can install {agent} in a custom base path other than `/Library`.  When installing {agent} with the `./elastic-agent install`
-command, use the `--base-path` CLI option to specify the custom base path.
+command, use the `--base-path` CLI option to specify the custom absolute base path.
```

Comment:
By "absolute," do you mean that variables are not supported?

Reply:
No, I meant it more in the sense of https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/